### PR TITLE
Added logging to be able to spot easily json comparison assertion fai…

### PR DESCRIPTION
**Pull request by israelcolomer**
…lures.

Added capability of asserting on response headers.

### CODE REVIEW

File: cucumber/src/main/java/org/chodavarapu/datamill/cucumber/FuzzyJsonTester.java [hunk 1]
```diff
@@ -4,11 +4,16 @@
...
 import org.json.JSONException;
 import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * @author Israel Colomer (israelcolomer@gmail.com)
  */
 public class FuzzyJsonTester {
+    private static final String ANY_VALUE = "ANY_VALUE";
```
Comment (rchodava) [inline cucumber/src/main/java/org/chodavarapu/datamill/cucumber/FuzzyJsonTester.java]: I think maybe we should change this to *

File: cucumber/src/main/java/org/chodavarapu/datamill/cucumber/FuzzyJsonTester.java [hunk 2]
```diff
@@ -52,8 +57,9 @@ private static boolean areJsonObjectsSimilarEnough(JSONObject expected, JSONObje
     }
 
     private static boolean isSimilar(Object expected, Object actual) {
+        logger.debug("Comparing EXPECTED {} and ACTUAL {}", expected, actual);
```
Comment (rchodava) [inline cucumber/src/main/java/org/chodavarapu/datamill/cucumber/FuzzyJsonTester.java]: Why caps EXPECTED and ACTUAL here?

**Comment by rchodava** [comment]
:+1: if over 70% ;)

**Comment by israelcolomer** [comment]
Shall we say from now on, no 👍  if no testing covering the commit(s) + coveralls not happy (> 70%)?

**Pull request merged**